morph_tool/utils.py

```diff
@@ -27,6 +27,7 @@ def neurondb_dataframe(filename: Path) -> pd.DataFrame:
     Args:
         filename: the neurondb.dat file
     '''
-    df = pd.read_csv(filename, sep=' ', names=['name', 'layer', 'mtype'])
+    df = pd.read_csv(filename, sep=r'\s+', names=['name', 'layer', 'mtype'],
+                     na_filter=False)
```

Comment:
What is the reason for `na_filter=False`? What would happen if NA value leaks in `neurondb` file?

Reply:
I had the same question: specifically; how big is the performance increase vs. will this mean that malformed/erroneous neurondb's are passed through silently?  Taking a perf. hit, and discover problems early is a tradeoff that is worth taking, unless we're sure that bad files will never happen (which is hard, since ppl edit these manually), or the perf increase is large.

Reply:
I saw it in morph-validator so I figured out it was a good idea but yes, after more thinking I'm not sure it's worthwhile. I will remove it.